models/user.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-    email: DataTypes.STRING,
+    email: {type: DataTypes.STRING, unique: true},
     password: DataTypes.STRING,
```

Comment:
Hey - is this actually being used to check a validation? I am just unfamiliar with the syntax.

Reply:
I still need to check and see if this works. It looks like it should...